impress-org/givewp
# Fix: update anonymous donation block placeholder UI/UX.

## Description

This PR simply updates the font styles associated with the Anonymous Donation block in the editor. The checkbox has also been marked as false so that the block can not be selected and toggled with a blue background. These changes provide consistency between the form labels styling and the behavior of the other block's with checkboxes in the editor view.

https://app.asana.com/0/1203547864809090/1205525808508497

## Affects
- Anonymous Donation block in the editor

## Visuals

<img width="1727" alt="Screen Shot 2023-09-19 at 2 33 49 PM" src="https://github.com/impress-org/givewp/assets/75056371/55fa1f6a-0926-45bd-b011-867dae9e977a">

## Testing Instructions

- Add the anonymous donation block.
- View in the editor that the label styles visually match the other field labels.
- Try selecting the checkbox within the block edit view - it will no longer flash blue inside.
- Verify it continues to work properly  in design mode and frontend 

## Pre-review Checklist




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205542539227769
